solidity/test/MockHyperlaneEnvironment.sol

```diff
@@ -4,25 +4,69 @@ pragma solidity ^0.8.13;
 import "../contracts/mock/MockOutbox.sol";
 import "../contracts/mock/MockInbox.sol";
 import "../contracts/AbacusConnectionManager.sol";
+import "../contracts/middleware/InterchainQueryRouter.sol";
+
+import {TypeCasts} from "../contracts/libs/TypeCasts.sol";
 
 contract MockHyperlaneEnvironment {
-    MockOutbox public outbox;
-    MockInbox public inbox;
+    MockOutbox public originOutbox;
+    MockInbox public originInbox;
+
+    MockOutbox public destinationOutbox;
+    MockInbox public destinationInbox;
 
-    mapping(uint32 => AbacusConnectionManager) connectionManagers;
+    mapping(uint32 => AbacusConnectionManager) public connectionManagers;
+    mapping(uint32 => InterchainQueryRouter) public queryRouters;
```

Comment:
why mix mappings and `originXXX` `destinationXXX`

Reply:
Yeah that's fair, i noticed that half way but was too lazy tbh

Reply:
Fixed it